packages/loader/container-utils/src/test/error.spec.ts

```diff
@@ -7,100 +7,18 @@
-            const originalError = { errorType: 3 };
-            const testError = CreateContainerErrorViaNormalize(originalError);
-
-            assert(testError.errorType === ContainerErrorType.genericError);
-        });
-        it("Should not expose original error props for telemetry besides message", () => {
-            const originalError: any = { hello: "world", message: "super important" };
-            const testError = CreateContainerErrorViaNormalize(originalError, { foo: "bar" });
-
-            assert(isILoggingError(testError));
-            assert(testError.getTelemetryProperties().hello === undefined);
-            assert(testError.getTelemetryProperties().message === "super important");
-        });
-        it("Should preserve the stack", () => {
-            const originalError = new Error();
-            const testError = CreateContainerErrorViaNormalize(originalError);
-
-            assert((testError as GenericError).stack === originalError.stack);
-        });
-        it("Should add errorType but drop telemetry props, as a new object", () => {
-            const loggingError = new LoggingError("hello", { foo: "bar" });
-            const testError = CreateContainerErrorViaNormalize(loggingError);
-
-            assert(testError.errorType === ContainerErrorType.genericError);
-            assert(isILoggingError(testError));
-            assert(testError.getTelemetryProperties().foo === undefined, "telemetryProps shouldn't be copied when wrapping");
-            assert(testError as any !== loggingError);
-        });
-
-        it("Should preserve telemetry props and existing errorType, and return same object", () => {
-            const loggingError = new LoggingError("hello", { foo: "bar" }) as LoggingError & { errorType: string };
-            loggingError.errorType = "someErrorType";
-            const testError = CreateContainerErrorViaNormalize(loggingError);
-
-            assert(testError.errorType === "someErrorType");
-            assert(isILoggingError(testError));
-            assert(testError.getTelemetryProperties().foo === "bar");
-            assert(testError as any === loggingError);
-        });
-        it("Check double conversion of generic error", async () => {
-            const err = {
-                message: "Test Error",
-            };
-            const error1 = CreateContainerErrorViaNormalize(err);
-            const error2 = CreateContainerErrorViaNormalize(error1);
-            assert.deepEqual(error1, error2, "Both errors should be same!!");
-            assert.deepEqual(error2.message, err.message, "Message text should not be lost!!");
-        });
-    });
     describe("DataProcessingError.create", () => {
         it("Should yield a DataProcessingError", () => {
-            const dpe = DataProcessingError.create("Some message", "someCodepath");
+            const dpe = DataProcessingError.create("Some message", "someCodepath", undefined, { someProp: 1234 });
```

Comment:
What type are we expecting here? For me it's hard to figure out without actually going into the code.